### PR TITLE
LPS-194276 - Refactor Automation | Workflow | Master

### PR DESCRIPTION
**Pull request by thaiscabrals**
Ticket: [LPS-194276](https://liferay.atlassian.net/browse/LPS-194276)
Test: WorkflowMetricsWorkloadByStep#SumOfPendingItemsGreaterThanTotalPendingWhenArePareallelBranches

### CODE REVIEW

**Comment by thaiscabrals** [comment]
ci:test:sf

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsWorkloadByStep.testcase [hunk 1]
```diff
@@ -246,43 +246,49 @@ definition {
 		property test.liferay.virtual.instance = "false";
 		property test.run.type = "single";
 
-		ApplicationsMenu.gotoPortlet(
-			category = "Workflow",
-			panel = "Applications",
-			portlet = "Process Builder");
+		task ("Given a workflow with parallel branches") {
```
Comment (patriciajeanperez) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsWorkloadByStep.testcase:249]: @thaiscabrals You can split setup into `Given.. and Given.. and Given` if needed

**Review by patriciajeanperez — CHANGES_REQUESTED** [review]

**Comment by thaiscabrals** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**